docs/pipeline-configuration-files/provisioning.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Provisioning'
+sidebar_position: 1
+---
+
+## Pipelines Directory
+
+When Conduit starts, it will search for all files ending with `.yml` or `.yaml` in folder `./pipelines` and its
+sub-folders. It will parse the configuration files and provision the pipelines defined in them.
+
+If you want Conduit to search a different directory, or want to provision a single file, you can specify the CLI flag
```

Comment:
This is implying that you can point `-pipelines.path` to a single file? Is that the case?

Reply:
Yup, that's right!

Reply:
yeah I tested it myself and came back to just delete this.